prestoadmin/package.py

```diff
@@ -74,6 +74,10 @@ def deploy_action(local_path, rpm_action):
 
 
 def deploy(local_path=None):
+    if not os.path.isfile(local_path):
```

Comment:
Should we verify that it's readable too? That seems like a common enough case that it shouldn't result in a stack trace for the user.